manifests/init.pp

```diff
@@ -235,6 +236,15 @@
       }
     }
 
+    if $firewall_backend {
+      augeas {
+        'firewalld::firewall_backend':
```

Comment:
This is only valid in firewalld >= 0.6.0 and will cause an error in the logs about unsupported options on versions less than that. Take this, it's dangerous to go alone https://github.com/voxpupuli/puppet-firewalld/pull/255

Reply:
From a style perspective this should read like

```puppet
augeas { 'firewalld::firewall_backend':
  changes => ["set FirewallBackend \"${firewall_backend}\"",],
}
```